### ISSUE TITLE
Frontmatter does not always parse properly in obsidian

### ISSUE THREAD

**Issue by lzilioli** [wormi4ok/evernote2md#111]
**Problem statement**

Some of my notes, when exported, their frontmatter fails to parse in obsidian, and thus, I am not able to run dataview queries as desired.

Here is an example:

```
---
date: '2014-04-30 03:55:32 +0000'
updated_at: '2014-04-30 03:55:32 +0000'
title: Fwd: Clever Household Hints.....
tags: [ 'archive', 'email spam' ]
source: mail.smtp

---
```

I found that if I go through and add double or single quotes to the title, obsidian immediately parses the frontmatter properly, and dataview is able to index it.

Is there a way to update the `--addFrontMatter` flag so that it surrounds the title in quotes? (will also need to escape whichever quote you surround with. e.g. `Fwd: 10 things that make you say "thats crazy"` should be exported as `title: "Fwd: 10 things that make you say \"thats crazy\""`

**Example file**

**Comment by wormi4ok** [comment]
Hey @lzilioli , thanks for reporting this issue. It shouldn't be a problem to fix this bug - I'll try to find time during upcoming holidays to submit a fix. 👍🏻

**Comment by wormi4ok** [comment]
Just published a new release - [v0.21.0](https://github.com/wormi4ok/evernote2md/releases/tag/v0.21.0) which includes a fix for this issue. Hope it helps!

**Comment by lzilioli** [comment]
This is great, thank you @wormi4ok
